BO4E-dotnet/ENUM/Fernschaltung.cs

```diff
@@ -0,0 +1,12 @@
+namespace BO4E.ENUM
+{
+
+    /// <summary>Fernschaltung</summary>
+    public enum Fernschaltung
+    {
+        /// <summary>Z06: vorhanden</summary>
```

Comment:
Andere ENUM-Werte sind immer alle in Großbuchstaben. ggf. der Konsistenz wegen anpassen?